shared/enclave/server/views/pin.ts

```diff
@@ -0,0 +1,246 @@
+import { BrowserWindow, ipcMain, IpcMessageEvent } from 'electron';
+
+const EVENT = 'enclave:pin';
+const scriptNonce = Math.floor(Math.random() * 1000000000000);
+const html = `
+<html>
```

Comment:
If anyone can think of a reasonable way to put this into its own HTML file and require it without having some webpack HTML handler, I'm all ears.

Reply:
Is node `fs` disabled here? Could just read it in as a text file